Automattic/wp-calypso
# Framework: Include HelloVote notice in Layout

This pull request seeks to add a new notice to encourage users to manage a yet-to-be-added setting for controlling a vote prompt on their site. The included text and design are very much unfinished and feedback or patches are welcome.

![Notice](https://cloud.githubusercontent.com/assets/1779930/19085663/1ac46812-8a39-11e6-9e30-cd7b5ae98156.png)

The intent of the notice is that it be shown only once, and that the user's preference to dismiss it be remembered permanently. This is also a limited-time component, expected to be removed in approximately one month.

__Testing Instructions:__

Verify that the notice is shown when navigating to any site-specific section, and that it is dismissed when clicking either the dismiss button or the Opt In action. The action should navigate you to the site settings screen.

If you need to reset your preference, you should navigate to [the REST API console](https://developer.wordpress.com/docs/api/console/), enter `POST /me/settings` as the URL, and `calypso_preferences={"helloVoteNoticeDismissed":null}` as the body.

cc @nylen
